fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/FDBRecordStore.java

```diff
@@ -350,6 +353,15 @@ public RecordMetaDataProvider getMetaDataProvider() {
         return metaDataProvider;
     }
 
+    /**
+     * Get the provider for the record store's meta-data.
+     * @return the meta-data source to use
+     */
+    @Nonnull
+    public Boolean getVersionChanged() {
```

Comment:
Should this be a `boolean`? It looks like the member variable is already

Reply:
done